invenio_cli/helpers/scripts.py

```diff
@@ -154,7 +156,10 @@ def update_statics(dev, loglevel, logfile):
         shutil.copyfile(src_file, dst_path / 'logo.svg')
 
     else:
-        pass
+        # dst_path is a path inside the container.
+        dst_path = '/opt/invenio/var/instance/static/images'
+        app_name = app_name.replace("-", "")
```

Comment:
`app_name` is `project_name` here and `project_name` may contain spaces too. `project_shortname` has spaces replaced with  `-`. 

Why are dashes problematic?

Reply:
My bad, on semantics. `project_name` is `os.path.basename(context)` so in the end is `project_shortname`.

What about `app_name` all over?

about dashes, is just the way docker creates the containers :/ it removes the dashes so we need to comply with it...